util/partition_integers

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+# partition_integers -- Partitions integer range into even ranges
+##
+import sys
+
+M = int(sys.argv[1])
+N = int(sys.argv[2])
+
+k = float(M)/N
```

Comment:
Yes, it will produce the correct answer. Could it better to set N as M is N>M?

On Feb 1, 2016, at 12:22 AM, Jaeho Shin <notifications@github.com<mailto:notifications@github.com>> wrote:


In util/partition_integers<https://github.com/HazyResearch/deepdive/pull/473#discussion_r51389627>:

> @@ -0,0 +1,16 @@
> +#!/usr/bin/env python
> +# partition_integers -- Partitions integer range into even ranges
> +##
> +import sys
> +
> +M = int(sys.argv[1])
> +N = int(sys.argv[2])
> +
> +k = float(M)/N


Thanks for catching this. I wasn't careful enough to include unit tests for this. I'll make sure it produces error on such case. Yes if we want more partitions than the number of elements, it'll end up with something like that. I think it'll produce the a correct partition that covers all elements, right?

—
Reply to this email directly or view it on GitHub<https://github.com/HazyResearch/deepdive/pull/473/files#r51389627>.